en/ros2/px4_ros2_navigation_interface.md

```diff
@@ -0,0 +1,245 @@
+# PX4 ROS 2 Interface Library: Navigation
+
+:::warning Experimental
+At time of writing parts of the PX4 ROS 2 Interface Library is experimental, and hence subject to change.
+:::
+
+The PX4 ROS 2 Interface Library for navigation enables developers to send their position estimates to PX4 directly using ROS 2. The interface provides a layer of abstraction from PX4 and the uORB messaging framework, and introduces a few sanity checks on the requested state estimation updates sent via the interface. These estimates are then fused into the EKF as internal PX4 estimates would.
```

Comment:
It doesn't "matter, but I quite like sentence line breaks. They make reviewing and commenting easier.

```suggestion
The PX4 ROS 2 Interface Library for navigation enables developers to send their position estimates to PX4 directly using ROS 2.
The interface provides a layer of abstraction from PX4 and the uORB messaging framework, and introduces a few sanity checks on the requested state estimation updates sent via the interface.
These estimates are then fused into the EKF as internal PX4 estimates would.
```